app/controllers/application.js

```diff
@@ -5,7 +5,9 @@ const { getOwner } = Ember;
 
 export default Ember.Controller.extend({
 
-  appVersion: config.APP.VERSION,
+  appVersion: Ember.computed(function(){
```

Comment:
instead this add `isMobileApp: config.cordova.enabled`

Reply:
changes are already done.